sql/item_mods.sql

```diff
@@ -5351,7 +5351,6 @@ INSERT INTO `item_mods` VALUES (11613,29,1);
 INSERT INTO `item_mods` VALUES (11614,1,5);
 INSERT INTO `item_mods` VALUES (11614,2,25);
 INSERT INTO `item_mods` VALUES (11614,5,-40);
-INSERT INTO `item_mods` VALUES (11614,162,5);
```

Comment:
For anyone looking to replace the bad mod that was removed from ardor pendant: it would be the kind that buffs drg's wyvern breath attacks and possibly blu magic breaths, not damage reduction which is what was on it. We have a mod for wyvern breath only in trunk so far, nothing that buffs any other breath damage.